pymatgen/core/structure.py

```diff
@@ -764,6 +764,28 @@ def get_neighbors(self, site, r, include_index=False):
                                       include_index=include_index)
         return [d for d in nn if site != d[0]]
 
+    def get_nearest_site(self, coords, site, r = None):
```

Comment:
Ok. I think if you use coord_utils pbc_shortest_vector does the same thing and is probably faster?